tasks/freeradius.yml

```diff
@@ -6,7 +6,7 @@
   become: true
   when:
     - ansible_distribution|string == 'Ubuntu'
-    - ansible_distribution_release|string == 'focal'
+    - ansible_distribution_release|string == 'focal' or ansible_distribution_release|string == 'jammy'
```

Comment:
Freeradius package, currently not available for `Ubuntu 22.04 (Jammy Jellyfish)`. 
See this :point_right:  https://networkradius.com/packages/